documentation/_puppetdb_nav.html

```diff
@@ -4,6 +4,7 @@ <h2 id="puppetdb">PuppetDB {{ page.doc.version }}</h2>
   <li><strong>General information</strong>
     <ul>
       <li><a href="{{puppetdb}}/index.html">Overview and requirements</a></li>
+      <li><a href="contributing.html">Contributing to PuppetDB</a></li>
```

Comment:
I'm not entirely sure if this is the correct way to link to the markdown doc. The other docs listed here seem to be within the `documentation` folder... which must be what `{{puppetdb}}` is referencing?